src/targeting.js

```diff
@@ -464,6 +464,14 @@ export function newTargeting(auctionManager) {
       if (typeof filterFunction === 'function') {
         bidsReceived = bidsReceived.filter(bid => latestAuctionForAdUnit[bid.adUnitCode] === bid.auctionId || !!filterFunction(bid))
       }
+
+      bidsReceived = bidsReceived
+        .map(bid => {
```

Comment:
this looks more suited to `forEach`. Also I think it should be done after all filtering is done, around line 480, or do you see a reason why it's better to tag them here?

Reply:
You're right, `forEach` is mor suited for this.

Yes, I agree that the property should be set after all filtering is done, provided it should be set on all the bids in the pool (as you mentioned in your comment below). The reason I added my change where I did was to avoid increasing the size of bid objects in the cases where `auctionId` would have the same value as the new property. But I think it's a good idea to set the property on all bids in the pool.